### ISSUE TITLE
[Experiment] Make global CTA a 90% experiment; localize.

### ISSUE THREAD

**Issue by ejregithub** [mozilla/bedrock#6792]
Reference the original work in #6629 

| Estimates |  |
| ------------- | ------------- |
| TL | 1 hour |
| PM | 1/2 day |
| ENG | 1 day |
| SEO | |
| ANA | ? |
| DES | |
| COPY | |

Description:  Notch up the global CTA to a 90% experiment and localize it in DE/FR. (Original description: Implement a test showing a Fx Account CTA in the Global Navigation to users visiting mozilla.org w. a Firefox browser instead of the current 'Download Firefox' CTA.)

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] Test is set to 90% of traffic and live by date TBD
- [ ] Localized strings provided (by FxA marketing) are enabled in DE and FR 
- [ ] Firefox version changed to >=57

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [x] Localization (Agency or Community) _Resolved: we're asking requester for localized strings_

#### Tasks
- [ ]

**Comment by ejregithub** [comment]
I think the reasoning may (may) be that users  with older versions may (may) not be able to update further, but can sign up for an account regardless of their version and our overarching goal is to promote Accounts. @hoosteeno - is that accurate - we're looking promote accounts to all Fx users?

**Comment by hoosteeno** [comment]
> I believe the ask here is to expand this experiment to 2 further locales (de and fr), as opposed to open this up for l10n? Generally we will open things up only once an experiment proves successful.

I clarified above. I asked the FxA team to provide localized strings.

> Also, I seem to remember an ask in Slack of opening this up to FF > 57 visitors which isn't yet detailed here. Is that still a requirement? (and also why 57, considering those users are pretty out of date?)

Getting people on Quantum puts them on solid footing for a whole host of things we care about that relate to user satisfaction and a deepening relationship -- accounts features like send tabs, UI improvements, better performance, pocket integration, and of course any security fixes in the interval. But going forward from Quantum, we should not assume that the website is the best place to convince people to update Firefox. In fact, product features should do this work. We can do more impactful things with the high-value web real estate we've previously allocated to promoting upgrades.

**Comment by alexgibson** [comment]
Thanks for the context re: 57. I think for an experiment this is fine, and I totally agree that trying hard to get people to update is a difficult task in terms of the website and the [UX problems](https://github.com/mozilla/bedrock/issues/6726) it presents. 

Should this become a long term feature, perhaps we could consider not using a hard coded version number though. Instead we could perhaps say, if the user is <= x major versions out of date, show FxA, else: show the download button.

**Comment by alexgibson** [comment]
Blocking this issue on strings.

**Comment by LaurenNiolet** [comment]
Are the l10n strings preventing us from exposing this to 90% of the audience in EN?

**Comment by alexgibson** [comment]
> Are the l10n strings preventing us from exposing this to 90% of the audience in EN?

Good suggestion 👍 If we're blocked on copy, let's split this out into two pieces of work.

**Comment by ejregithub** [comment]
@amychurchwell - ready for your code review, please.

**Comment by ejregithub** [comment]
@nohno 

The [committed file](https://github.com/mozilla/bedrock/blob/b31f5deecd41f98ef018684f4b57a36496349bbb/bedrock/base/templates/includes/protocol/navigation/index.html) lists:

"Create a Firefox Account"
and
"Check out the Benefits"

**Comment by nohno** [comment]
Hi!  The strings have been added here: https://docs.google.com/document/d/1qeGA50axmClm03KEC7opuF0DOgExyCj03rFd0l57rco/edit

@ejregithub  @LaurenNiolet

**Comment by ejregithub** [comment]
EN | DE | FR
-- | -- | --
Get a Firefox Account | Firefox-Konto erstellen | Créez un compte Firefox
Check out the Benefits | Das sind deine Vorteile | Découvrez les avantages

**Comment by alexgibson** [comment]
I've opened a PR to expand this experiment for `de` and `fr` above. 

Questions for @hoosteeno and/or @sghoseWI:

- Are we happy showing the experiment to the same population (i.e. 90%) for both German and French locales?
- Do we need any additional tracking in the utm params that we pass through to the FxA server for these locales, or can we simply pass through the same values that we're using for `en-US`?

**Comment by hoosteeno** [comment]
90% is good in de/fr.

Amplitude can segment by lang and locale so we shouldn't need to send that info.

**Comment by alexgibson** [comment]
Ok, this is now ready to go

**Comment by sghoseWI** [comment]
Thanks @alexgibson -- Would you be able to point me to a list of all the working UTM parameters for this experiment (control vs. experimental groups)? I'm hoping to use our BigQuery datasets to get more fine-grained data than what we can get from Amplitude alone.

**Comment by sghoseWI** [comment]
Experimental cohort sample results look good so far!  https://analytics.amplitude.com/mozilla-corp/chart/rghy2p9/edit/3wa0gki
